Google Scholar.js

```diff
@@ -742,8 +750,7 @@ ItemFactory.prototype.repairCitelet = function () {
 ItemFactory.prototype.repairTitle = function () {
 	// All-caps words of four or more characters probably need fixing.
 	if (this.v.title.match(/(?:[^a-z]|^)[A-Z]{4,}(?:[^a-z]|$)/)) {
-		this.v.title = ZU.capitalizeTitle(this.v.title.toLowerCase())
-						.replace(/([^0-9a-z])V([^0-9a-z])/, "$1v$2");
+		this.v.title = ZU.capitalizeTitle(this.v.title.toLowerCase(), true);
```

Comment:
Idk about cases, but I thought we were going to switch over to sentence case when fixing all-caps. Though in this case you're not really checking if everything is all-caps and it might be more correct to convert to title case for cases since they tend to contain a lot of proper nouns. Anyway, you'll have to make a call on that.